config/vufind/EDS.ini

```
@@ -418,6 +418,11 @@ AU = None
 ; useQuery above.
 ;useQueryMinLength = 3
 
+; Match a database names if its start is a close match to the
+; search query, off by at most this levenshtein distance.
```

This is usually capitalized:

```suggestion
; search query, off by at most this Levenshtein distance.
```